packages/cli/src/cli/commands/documents.ts

```diff
@@ -114,3 +114,9 @@ export const generatePreviewLink = errorHandler<GeneratePreviewParam>(
     logger.log(chalk.green(buildLink));
   },
 );
+export const DOCUMENT_EXAMPLES = [
+  {
+    description: "Generate preview link for given document ID",
+    command: "$0 document preview 8MwijBYyp3B41slkdfjalkdfdziXkjyynTREdst8FauQ",
```

Comment:
Use a fake id.

Reply:
I copied the original one and made changes to ensure it doesn't exist.
Anyways, I will update it so it explicitly appears like a fake id.